lib/pronto/config.rb

```diff
@@ -52,27 +52,57 @@ def bitbucket_hostname
     end
 
     def warnings_per_review
-      ENV['PRONTO_WARNINGS_PER_REVIEW'] && Integer(ENV['PRONTO_WARNINGS_PER_REVIEW']) || @config_hash['warnings_per_review']
+      fetch_integer('warnings_per_review')
     end
 
     def max_warnings
-      ENV['PRONTO_MAX_WARNINGS'] && Integer(ENV['PRONTO_MAX_WARNINGS']) || @config_hash['max_warnings']
+      fetch_integer('max_warnings')
     end
 
     def message_format(formatter)
       formatter_config = @config_hash[formatter]
       if formatter_config && formatter_config.key?('format')
         formatter_config['format']
       else
-        ENV['PRONTO_FORMAT'] || @config_hash['format']
+        fetch_value('format')
       end
     end
 
+    def skip_runners
+      fetch_list('skip_runners')
+    end
+
+    def runners
+      fetch_list('runners')
+    end
+
     def logger
       @logger ||= begin
-        verbose = ENV['PRONTO_VERBOSE'] || @config_hash['verbose']
+        verbose = fetch_value('verbose')
         verbose ? Logger.new($stdout) : Logger.silent
       end
     end
+
+    private
+
+    def fetch_integer(key)
+      env_key = env_key(key)
+
+      (ENV[env_key] && Integer(ENV[env_key])) || @config_hash[key]
+    end
+
+    def fetch_value(key)
+      ENV[env_key(key)] || @config_hash[key]
+    end
+
+    def env_key(key)
+      "PRONTO_#{key.upcase}"
+    end
+
+    def fetch_list(key)
+      Array(fetch_value(key)).flat_map do |runners|
+        runners.split(' ')
```

Comment:
I'm not sure what is the most common separator for CLI tools.
How about using `,` instead?

Then quotes can be omitted in the shell command:

```bash
PRONTO_RUNNERS=rubocop,reek pronto run
```

Reply:
changed to `,`